src/main/java/org/testng/ClassMethodMap.java

```diff
@@ -17,26 +17,26 @@
  * @author <a href='mailto:the[dot]mindstorm[at]gmail[dot]com'>Alex Popescu</a>
  */
 public class ClassMethodMap {
-  private Map<Object, List<ITestNGMethod>> m_classMap = Maps.newHashMap();
+  private Map<Object, List<ITestNGMethod>> classMap = Maps.newHashMap();
   // These two variables are used throughout the workers to keep track
   // of what beforeClass/afterClass methods have been invoked
-  private Map<ITestClass, Set<Object>> m_beforeClassMethods = Maps.newHashMap();
-  private Map<ITestClass, Set<Object>> m_afterClassMethods = Maps.newHashMap();
+  private Map<ITestClass, Set<Object>> beforeClassMethods = Maps.newHashMap();
```

Comment:
By the way, why that renaming of the field? This should be a separate PR (assuming we want to do that at all).